.github/workflows/tests_emulator.yml

```
@@ -25,6 +20,13 @@ jobs:
     timeout-minutes: 75
     env:
       CODECOV_TOKEN: ${{ secrets.CODECOV_TOKEN }}
+      TEST_RELEASE_BUILD: true
+      # could be better, '/home/runner' should be '~/' but neither that nor '$HOME' worked
```

I don't clearly see where this should be pointing, but consider that these might be an answer:
```yml
env:
  STOREFILEDIR: ${{ gitHub.workspace }}
  STOREFILEDIR: ${{ runner.temp }}/SRC
```

---

we have, since Android 1 (when AnkiDroid started) have these values hard-coded in build.gradle, this is more of an extension where now (finally!) we can *also* optionally take environment variables and un-pin the file location if they are specified. If any of key alias, store location, store pass, key pass are not specified they'll fall back to the hard-coded values

At any rate, that comment is now actually vestigial, it was me not realizing that bash interpolation (~) and environment variables (`$whatever`) don't work in env blocks, only workflow variables do. And we're handling it in a different way now

I'll clean the stale comment out at some point...